torch/fx/experimental/test_rewriter.py

```diff
@@ -0,0 +1,51 @@
+import torch
+from torch.fx import symbolic_trace
+from torch.testing._internal.jit_utils import JitTestCase
+from torch.testing._internal.common_utils import run_tests
+
+
+class TestASTRewriter(JitTestCase):
+
+    def test_assert_no_msg(self):
+        class M(torch.nn.Module):
+            def forward(self, a, b):
+                assert a == b
+                return a + b
+        m = M()
+        traced = symbolic_trace(m)
+        traced.graph.lint(traced)
```

Comment:
I think for all these tests you would want to do two additional things:
1. Check the IR to make sure there's a `call_function` node with `target == torch.Assert`
2. Ensure that when the code is run, the assert throws when it's supposed to and doesn't throw when it's not supposed to. You can use `self.assertRaisesRegex` to intercept assertions and verify that they are thrown